cmake/llext-edk.cmake

```diff
@@ -206,6 +210,19 @@ function(edk_write_var target var_name var_value)
         # Each element of the list is wrapped in quotes and is separated by a space.
         list(JOIN exp_var_value "\" \"" exp_var_value_str)
         file(APPEND ${edk_file_${target}} "${var_name} = \"${exp_var_value_str}\"\n")
+    elseif(target STREQUAL "PYTHON")
+        # Python: export lists of f-strings of the form:
+        #
+        #   var = [f"value1", f"value2", ...]
+        #
+        edk_escape(${target} "${exp_var_value}" exp_var_value)
+        set(DASHIMACROS "-imacros{${install_dir_var}}/")
+        set(DASHI "-I{${install_dir_var}}/")
+        string(CONFIGURE "${var_value}" exp_var_value @ONLY)
+        # Each element of the list is wrapped in f-quotes and is separated by a
+        # space and a comma.
+        list(JOIN exp_var_value "\", f\"" exp_var_value_str)
+        file(APPEND ${edk_file_${target}} "${var_name} = [f\"${exp_var_value_str}\"]\n")
```

Comment:
this doesn't look like yaml to me :wink: 
ref: https://github.com/zephyrproject-rtos/zephyr/pull/78727#discussion_r1768662769

> then a yaml file would have been just as easy / nice to use 😉
>
> > Only additional file format I consider acceptable is yaml

remember, yaml can be read directly into Python dict and is much nicer to work with.
We actually have a yaml module in Zephyr CMake now :smile: